assembly/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.eclipse.rdf4j</groupId>
 		<artifactId>rdf4j</artifactId>
-		<version>3.3.0-SNAPSHOT</version>
+		<version>3.3.0-M1-SNAPSHOT</version>
```

Comment:
I wouldn't use milestone markers in the SNAPSHOT versions in the pom if I were you. Stick with just 3.3.0-SNAPSHOT. Only use "3.3.0-M1" for the actual tagged version.